TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/nonRoadRunner/BackupShoot.java

```diff
@@ -0,0 +1,159 @@
+    private DcMotor frontLeftDrive = null;
+    private DcMotor frontRightDrive = null;
+    private DcMotor backLeftDrive = null;
+    private DcMotor backRightDrive = null;
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        frontLeftDrive  = hardwareMap.get(DcMotor.class, Constants.leftFrontDriveMotor()); //0
+        frontRightDrive = hardwareMap.get(DcMotor.class, Constants.rightFrontDriveMotor()); //3
+        backLeftDrive = hardwareMap.get(DcMotor.class, Constants.leftBackDriveMotor()); //1
+        backRightDrive = hardwareMap.get(DcMotor.class, Constants.rightBackDriveMotor()); //2
+
+        waitForStart();
+
+        frontLeftDrive.setDirection(DcMotor.Direction.REVERSE);
+        frontRightDrive.setDirection(DcMotor.Direction.FORWARD);
+        backLeftDrive.setDirection(DcMotor.Direction.REVERSE);
+        backRightDrive.setDirection(DcMotor.Direction.FORWARD);
+
+        frontLeftDrive.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        frontRightDrive.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        backLeftDrive.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        backRightDrive.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+
+        frontLeftDrive.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        frontRightDrive.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        backLeftDrive.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        backRightDrive.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        double FLposition = frontLeftDrive.getCurrentPosition();
+        double FRposition = frontRightDrive.getCurrentPosition();
+        double BLposition = backLeftDrive.getCurrentPosition();
+        double BRposition = backRightDrive.getCurrentPosition();
+
+        backRightDrive.setTargetPosition(1000);
+        backLeftDrive.setTargetPosition(1000);
+        frontRightDrive.setTargetPosition(1000);
+        frontLeftDrive.setTargetPosition(1000);
+
+        backRightDrive.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        backLeftDrive.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        frontRightDrive.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+        frontLeftDrive.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+
+        backRightDrive.setPower(.5);
+        backLeftDrive.setPower(.5);
+        frontRightDrive.setPower(.5);
+        frontLeftDrive.setPower(.5);
+
+        Shooter shooter = new Shooter(hardwareMap, telemetry);
+        Intake intake = new Intake(hardwareMap);
+
+        while(frontLeftDrive.isBusy() && frontRightDrive.isBusy() && backLeftDrive.isBusy() && backRightDrive.isBusy()) {
+            telemetry.addData("Positions", "frontLeft (%d), backLeft (%d), frontRight (%d), backRight (%d)", frontLeftDrive.getCurrentPosition(), frontRightDrive.getCurrentPosition(), backLeftDrive.getCurrentPosition(), backRightDrive.getCurrentPosition());
+            telemetry.update();
+        }
+        shooter.setMotorPower(0.7);
```

Comment:
@NormalDuck @Glowdisk, you'll need to add this `shooter.setMotorPower` and `intake.setCollectBalls()` & `instake.setStopCollecting()` in the 2939 repo